manifests/guestcluster/1.23/pvcsi.yaml

```diff
@@ -228,6 +228,10 @@ spec:
               value: {{ .PVCSINamespace }}
             - name: X_CSI_SERIAL_VOL_ACCESS_TIMEOUT
               value: 3m
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: 65532
+            runAsGroup: 65532
```

Comment:
I logged into guest cluster OS and observed following UID and GID

```
nobody:x:65534:65533:Unprivileged User:/dev/null:/bin/false
```

Reply:
articles [nobody - Ubuntu Wiki](https://wiki.ubuntu.com/nobody) and https://github.com/prometheus/prometheus/issues/3441#issuecomment-342982777 state that user ID 65534 is registered for NFS purpose only and should NOT be used for running any other process. Hence skipped using this particular user ID

Reply:
Thanks for the detail.